src/components/clusters/ClusterDetailsMultiRegion/components/MachinePools/components/EditMachinePoolModal/fields/WindowsLicenseIncludedField.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useField } from 'formik';
+
+import { Content, ContentVariants } from '@patternfly/react-core';
+
+import links from '~/common/installLinks.mjs';
+import { fieldId as instanceTypeFieldId } from '~/components/clusters/common/ScaleSection/MachineTypeSelection/MachineTypeSelection';
+import { CheckboxField } from '~/components/clusters/wizards/form';
+import ExternalLink from '~/components/common/ExternalLink';
+import PopoverHint from '~/components/common/PopoverHint';
+import { MachineType, NodePool } from '~/types/clusters_mgmt.v1';
+
+const fieldId = 'windowsLicenseIncluded';
+
+type WindowsLicenseIncludedFieldProps = {
+  isEdit?: boolean;
+  // Manually adding this field until backend api adds support to it -> https://issues.redhat.com/browse/OCMUI-2905
+  currentMP?: NodePool & { imageType?: string };
```

Comment:
Since the backend api haven't added support for this field yet, I adjusted the type to contain the new expected property manually.
(I do so on other places around the PR as well, marking each place with a todo comment linked to the related ticket [OCMUI-2905](https://issues.redhat.com/browse/OCMUI-2905))

The reason for adding the comment is to make the change of the type visible on the code and relate it to the feature's changes, so we won't forget where this type change came from